controllers/activemqartemis_reconciler.go

```diff
@@ -1668,11 +1671,16 @@ func (reconciler *ActiveMQArtemisReconcilerImpl) NewPodTemplateSpecForCR(customR
 	}
 
 	reqLogger.Info("Checking out extraMounts", "extra config", customResource.Spec.DeploymentPlan.ExtraMounts)
-	brokerPropertiesConfigMapName := reconciler.addConfigMapForBrokerProperties(customResource)
-	configMapsToCreate := []string{brokerPropertiesConfigMapName}
-	configMapsToCreate = append(configMapsToCreate, customResource.Spec.DeploymentPlan.ExtraMounts.ConfigMaps...)
 
-	extraVolumes, extraVolumeMounts := createExtraConfigmapsAndSecrets(container, configMapsToCreate, customResource.Spec.DeploymentPlan.ExtraMounts.Secrets)
+	configMapsToCreate := customResource.Spec.DeploymentPlan.ExtraMounts.ConfigMaps
+	secretsToCreate := customResource.Spec.DeploymentPlan.ExtraMounts.Secrets
+	resourceName, isSecret := reconciler.addResourceForBrokerProperties(customResource, namer)
```

Comment:
why do we need `isSecret`?

Reply:
it is for migration, an existing CR can still have a configMap, that will remain till there is some mod, the mount point is then different